src/protocol/client.h

```diff
@@ -77,19 +85,16 @@ protocolClientIoReadFd(ProtocolClient *const this)
 }
 
 /***********************************************************************************************************************************
-Functions
+Client Functions
 ***********************************************************************************************************************************/
-// Execute a command and get the result
-FN_EXTERN PackRead *protocolClientExecute(ProtocolClient *this, ProtocolCommand *command, bool resultRequired);
-
 // Move to a new parent mem context
 FN_INLINE_ALWAYS ProtocolClient *
 protocolClientMove(ProtocolClient *const this, MemContext *const parentNew)
 {
     return objMove(this, parentNew);
 }
 
-// Do not send exit command to the server when the client is freed
+// Do not send exit request to the server when the client is freed
```

Comment:
This is just for the very special ping case where it does not fully connect to the server. We don't want to send an exit request because it would not accepted.

However, the comment update here is not right since exit is a command. Got a little too happy with search/replace.

Reply:
Fixed this in b28552febcc4.